cmd/soroban-cli/src/log/event.rs

```diff
@@ -15,12 +13,13 @@ pub fn events(events: &[xdr::DiagnosticEvent]) {
 
         let _enter = span.enter();
 
-        let xdr = event.to_xdr_base64(xdr::Limits::none()).unwrap();
+        // Convert the event to JSON
         let json = serde_json::to_string(event).unwrap();
+
         if span.metadata().unwrap().level() == &Level::INFO {
-            info!("{i}: {xdr} {json}");
+            info!("{i}: {json}");
         } else {
-            debug!("{i}: {xdr} {json}");
+            debug!("{i}: {json}");
```

Comment:
This output is part of the logging and it was just recently written to output both the XDR and the JSON in:
- #1600

The issue being addressed by this PR is to change the stdout output of the `stellar contract events` subcommand, not the log output. If you see the original issue the original issue has an example of what the command is to run, and what the output looks like today. It's that output that the issue describes being changed from Rust debug format to JSON.